sky/spot/spot_utils.py

```diff
@@ -16,13 +16,14 @@
 from sky import sky_logging
 from sky.backends import backend_utils
 from sky.skylet import job_lib
+from sky.utils import common_utils
 from sky.utils import log_utils
-from sky.spot import constants
 from sky.spot import spot_state
 from sky.utils import subprocess_utils
 
 logger = sky_logging.init_logger(__name__)
 
+SPOT_CONTROLLER_NAME = f'sky-spot-controller-{common_utils.get_user_hash()}'
```

Comment:
Short comment to say: Add user hash so that two users don't have the same controller VM on shared-account clouds such as GCP.